ql/methods/finitedifferences/utilities/fdminnervaluecalculator.cpp

```diff
@@ -118,8 +118,7 @@ namespace QuantLib {
         Size direction)
     : FdmCellAveragingInnerValue(
         payoff, mesher, direction,
-        ext::function<Real(Real)>(static_cast<Real2RealFct>(std::exp))) {
-    }
+        ext::function<Real(Real)>([](Real x) { return std::exp(x); })) {}
```

Comment:
It should be possible to pass just the lambda in order to avoid an extra move construction of `ext::function` in the delegating constructor.

Also can you delete the typedef for `Real2RealFct` now that it's unused?

Besides that everything else looks good to me.

Reply:
Yes, that makes sense. We just pushed that.